apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -14,40 +14,82 @@ import no.nav.etterlatte.libs.common.behandling.BehandlingType
 import no.nav.etterlatte.libs.common.behandling.Persongalleri
 import no.nav.etterlatte.libs.common.behandling.RevurderingAarsak
 import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+import no.nav.etterlatte.libs.common.grunnlag.NyeSaksopplysninger
+import no.nav.etterlatte.libs.common.grunnlag.opplysningstyper.Opplysningstype
+import no.nav.etterlatte.libs.common.grunnlag.opplysningstyper.SoeknadMottattDato
+import no.nav.etterlatte.libs.common.gyldigSoeknad.GyldighetsResultat
+import no.nav.etterlatte.libs.common.gyldigSoeknad.VurderingsResultat
 import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
 import no.nav.etterlatte.libs.common.sak.Sak
+import no.nav.etterlatte.libs.common.tidspunkt.Tidspunkt
+import no.nav.etterlatte.libs.common.tidspunkt.toLocalDatetimeUTC
+import no.nav.etterlatte.libs.common.tidspunkt.toTidspunkt
+import no.nav.etterlatte.libs.common.toJsonNode
 import no.nav.etterlatte.oppgaveny.OppgaveServiceNy
-import no.nav.etterlatte.sak.SakDao
-import no.nav.etterlatte.sak.filterSakerForEnheter
+import no.nav.etterlatte.sak.SakService
 import org.slf4j.LoggerFactory
 import java.time.LocalDateTime
 
 class BehandlingFactory(
     private val oppgaveService: OppgaveServiceNy,
     private val grunnlagService: GrunnlagService,
     private val revurderingService: RevurderingService,
-    private val sakDao: SakDao,
+    private val gyldighetsproevingService: GyldighetsproevingService,
+    private val sakService: SakService,
     private val behandlingDao: BehandlingDao,
     private val hendelseDao: HendelseDao,
     private val behandlingHendelser: BehandlingHendelserKafkaProducer,
     private val featureToggleService: FeatureToggleService
 ) {
     private val logger = LoggerFactory.getLogger(this::class.java)
+
+    fun opprettBehandling(behov: NyBehandlingRequest): Behandling {
+        val soeker = behov.persongalleri.soeker
+
+        val sak = sakService.finnEllerOpprettSak(soeker, behov.sakType)
+
+        val behandling = opprettBehandling(
+            sak.id,
+            behov.persongalleri,
+            behov.mottattDato,
+            Vedtaksloesning.GJENNY
+        )
+
+        if (behandling == null) {
+            throw IllegalStateException("Kunne ikke opprette behandling")
+        }
+
+        val gyldighetsvurdering = GyldighetsResultat(
+            VurderingsResultat.KAN_IKKE_VURDERE_PGA_MANGLENDE_OPPLYSNING,
+            listOf(),
```

Comment:
Er det noe forskjell på å bruke denne kontra `emptylist()`?

Reply:
Skulle vært `emptyList()`. Copy-pastet bare fra gyldig-soeknad-appen. Lurer på om jeg skal ta litt sånn små-opprydding etter branchen er merget for å unngå at PR-en blir enda større.